modules/references/client/api/references.api.js

```diff
@@ -76,6 +76,29 @@ export async function read({ userFrom, userTo }) {
   );
 }
 
+/**
+ * API request: read references written by loggedIn user, filter them by userTo,
+ * and sort by 'created' field starting from the most recent date
+ *
+ * @param {string} userTo - id of user who received the reference
+ * @returns Promise<Reference[]> - array of the found references
+ */
+export async function readMine({ userTo }) {
+  const params = { userTo };
```

Comment:
Total nit-pick and feel free to ignore; might make sense to pass `userTo` directly to `get()` instead of assigning it to intermediate `params` which is used anyway just once?